docs/README.md

```diff
@@ -374,50 +247,60 @@ non-authenticated case and return a different HTML page, perhaps
 informing the client to try again with the proper secret.
 
 ```javascript
+// @demos/login.js
 let login = args => ({ html: `<html><body>please say the magic word.</body></html>` })
 ```
 
 The `if` combinator composes these three functions as you might
-expect. This example is bundled in the shell samples as
-[`@demos/if.js`](https://github.com/ibm-functions/shell/blob/master/app/demos/if.js).
+expect.
 
 ```javascript
+// @demos/if.js
 composer.if(
   /* cond */ 'authenticate',
   /* then */ 'welcome',
   /* else */ 'login')
 ```
 
+Now, enter the following in Shell to deploy and run the app.
+
 ```bash
+# enter in Shell
 # create required actions
-$ fsh action create authenticate @demos/authenticate.js
-$ fsh action create welcome @demos/welcome.js
-$ fsh action create login @demos/login.js
+> action create authenticate @demos/authenticate.js
+> action create welcome @demos/welcome.js
+> action create login @demos/login.js
 
 # create app
-$ fsh app create if @demos/if.js
+> app create if @demos/if.js
 
 # invoke app, with no secret parameter
-$ fsh app invoke if
+> app invoke if
 {
   html: "<html><body>please say the magic word.</body></html>"
 }
 
 # now invoke with secret parameter
-$ fsh app invoke if -p token secret -p name if-combinator
+> app invoke if -p token secret -p name if-combinator
 {
   html: "<html><body>welcome if-combinator!</body></html>"
 }
 ```
 
+_Tip:_ You can see the output data of an action node in the `Session Flow` graph by clicking on the node. This will bring you to the corresponding activation. _Note:_ An inline function node is not clickable as it does not generate any record about its execution. 
+
```

Comment:
Technically there is one. It’s the conductor that it ran as part of.

Reply:
I'll edit this. @tardieu We might need a separate doc that explains how `conductor` works in v2.